src/com/apeshko/javacore/task_02/Application.java

```diff
@@ -0,0 +1,63 @@
+package com.apeshko.javacore.task_02;
+
+public class Application {
+    public static void dynamicArrayTest() {
+        DynamicArray dynamicArray = new DynamicArray(2);
+
+        System.out.println(dynamicArray);
+
+        dynamicArray.add(1);
+        dynamicArray.add(2);
+
+        System.out.println(dynamicArray);
+
+        dynamicArray.add(3);
+        dynamicArray.add(4);
+
+        System.out.println(dynamicArray);
+
+        dynamicArray.add(5);
+        System.out.println(dynamicArray);
+
+        System.out.println(dynamicArray.get(0));
+        System.out.println(dynamicArray.get(4));
+
+        dynamicArray.removeAt(0);
+        dynamicArray.removeAt(3);
+        System.out.println(dynamicArray);
+    }
+
+    public static void flexibleDynamicArrayTest() {
+        FlexibleDynamicArray<String> flexibleDynamicArray = new FlexibleDynamicArray<String>(String[].class, 2);
```

Comment:
Why do we need provide class as a constructor param? we use generic param for the same purpose)